### PR TITLE
Two new jobs

### PR DESCRIPTION
**Pull request by JakeDaBoss**
Colony Doctor and Colony Officer.

Standalone ID cards, mining access, and their gear down on the colony so they don't take (in the best case) shipside supplies.

### CODE REVIEW

**Comment by MarshalRed** [comment]
This is good

**Comment by JakeDaBoss** [comment]
Offhand comment, but:
Fully tested, and works as intended on  my local dev branch.

File: code/game/objects/structures/ishimura/ishimura-signs.dm [hunk 1]
```diff
@@ -0,0 +1,31 @@
+/obj/structure/sign/ishimura
```
Comment (NanakoAC) [inline code/game/objects/structures/ishimura/ishimura-signs.dm:1]: this unrelated code shouldn't be here

File: code/_helpers/pixels.dm [hunk 2]
```diff
@@ -214,6 +214,10 @@
 	mover.modify_pixels(rejection*-1)
 
 
+// Currently something for Ishimura Signs.
```
Comment (NanakoAC) [inline code/_helpers/pixels.dm:217]: you've mixed up the signs branch into this one

File: code/modules/admin/verbs/adminhelp.dm [hunk 3]
```diff
@@ -36,7 +38,7 @@
 	for(var/original_word in msglist)
 		var/word = ckey(original_word)
 		if(word)
-			if(!(word in GLOB.adminhelp_ignored_words))
+			if(!(word in adminhelp_ignored_words))
```
Comment (NanakoAC) [inline code/modules/admin/verbs/adminhelp.dm:41]: this isn't adminhelp related either

File: html/changelogs/.all_changelog.yml [hunk 4]
```diff
@@ -564,6 +564,10 @@ DO NOT EDIT THIS FILE BY HAND!  AUTOMATICALLY GENERATED BY ss13_genchangelog.py.
   - tweak: Increased the standard device powercell to hold 50 max charge instead of
       25. Can now be used in stunprods and likeminded devices.
 2020-08-09:
+  Nanako:
+  - rscadd: Flooring is now multilayered, and on the outer level of the ship, includes
```
Comment (NanakoAC) [inline html/changelogs/.all_changelog.yml:568]: changelogs don't belong here

**Review by NanakoAC — CHANGES_REQUESTED** [review]
Initiative is commendable, but snype did ask you to wait. Lets get his input on this

On another note, see commented changes, several other unrelated branches have gotten mixed into this, it looks like you need to rebase it to dev

**Comment by JakeDaBoss** [comment]
I'm bouncing this PR.

Not only is the entire branch FUBAR, I'm also going to be including this into my dynamic mapsys fixes, and do it right the first time.
